.github/workflows/release.yaml

```diff
@@ -48,3 +49,11 @@ jobs:
           GORELEASER_KEY: "${{ secrets.GORELEASER_KEY }}"
           GEMFURY_PUSH_TOKEN: "${{ secrets.GEMFURY_PUSH_TOKEN }}"
           SNAPCRAFT_STORE_CREDENTIALS: "${{ secrets.SNAPCRAFT_STORE_CREDENTIALS }}"
+      - name: "Notify Slack of goreleaser status"
+        if: "always()"
+        uses: "8398a7/action-slack@v3"
```

Comment:
🤔 My first reaction was "Is this a trustworthy GitHub action?".

I found the [official Slack](https://github.com/marketplace/actions/slack-send-to-slack) GitHub Action, which is verified. Mind sharing why you used action-Slack instead of the official action?

Reply:
the 8398a7/action-slack is a very popular action. In the past, I've used it because it allows for detailed customisation that might become handy. A quick search shows that projects such as Laravel, Cilium among others, use it, including this [GitHub repo](https://github.com/github/codeql/blob/2f2c9f8943ef8df2fa8472f0946f0d5a4ccb3475/actions/ql/lib/ext/manual/8398a7_action-slack.model.yml#L4)
That said, happy to switch it to the one you linked here. Will update shortly!

Reply:
All good, it was more like I wasn't sure I could trust that action, and if there was any specific features one provided the other didn't, I'd typically favour the official action for security reasons.